doc/api/util.md

```diff
@@ -1593,6 +1593,36 @@ $ node negate.js --no-logfile --logfile=test.log --color --no-color
 { logfile: 'test.log', color: false }
 ```
 
+## `util.parseEnv(content)`
```

Comment:
I'm wondering why this is useful. It seems to open a new attack vector rather than being something people would use instead of `util.parseEnvFile`. It's not a blocker, but what's the reasoning behind this API?

Reply:
People that has environment variables encrypted somewhere and want to use dotenv parser can use this function to get an object where they can later assign it into process.env. Previous reviews by @GeoffreyBooth includes the motivation for this.